devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -79,13 +79,13 @@ metadata {
 
 		// Yale siren only
 		preferences {
-			input name: "alarmLength", type: "number", title: "Alarm length", description: "This setting does not apply to all devices", range: "1..10"
+			input name: "alarmLength", type: "number", title: "Alarm length", description: "Enter alarm length", range: "1..10"
```

Comment:
Was this supposed to be "This setting only applies to Yale sirens."?

Reply:
check the comment on the Jira ticket